tests/unit/test_charm.py

```diff
@@ -393,44 +393,48 @@ def test_datasource(
     "files, expected",
     [
         (
-            [
+            (
                 "/layers/paketo-buildpacks_bellsoft-liberica/jre/bin/java",
                 "/workspace/org/springframework/boot/loader/JarLauncher.class",
                 "/workspace/BOOT-INF/lib/mysql-connector-j-10.1.1.jar",
-            ],
+            ),
             True,
         ),
         (
-            [
+            (
                 "/layers/paketo-buildpacks_bellsoft-liberica/jre/bin/java",
                 "/workspace/org/springframework/boot/loader/JarLauncher.class",
-            ],
+            ),
             False,
         ),
     ],
 )
 def test_buildpack_app_mysql_capability(
-    harness: Harness, patch: SpringBootPatch, files: list[str], expected: bool
+    harness: Harness, patch: SpringBootPatch, files: set[str], expected: bool
```

Comment:
Shouldn't it be `tuple` rather than `set`? A set constructor would use `{...}`

Reply:
Fixed